### PR TITLE
Add GSSICB COH06 config and service metadata files

### PR DESCRIPTION
**Pull request by hjkristenson**
Config files have been generated for all of the COH06 services, and they were used with the print_egis_metadata.py script to output service-specific metadata files for each config.

### CODE REVIEW

**Comment by hjkristenson** [comment]
There is a lot of common overlap for the config files. There may be a more efficient way to deal with this. It may also be prudent to write a script that generates a config file (to reduce the change of copy-paste errors) and perhaps we could roll that right into generating metadata from the freshly generated config file.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.6.1]
+### Added
+- Config files and service metadata files for all COH06 services
+
 ## [0.6.0]
 ### Added
 - Scripts to generate GSSICB Coherence image services
```
Comment (jacquelynsmale) [inline CHANGELOG.md:14]:
```suggestion
- Scripts to generate GSSICB Coherence image services
- Config files and service metadata files for all COH06 services
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.6.1]
```
Comment (jacquelynsmale) [inline CHANGELOG.md:8]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+### Added
```
Comment (jacquelynsmale) [inline CHANGELOG.md:9]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Config files and service metadata files for all COH06 services
```
Comment (jacquelynsmale) [inline CHANGELOG.md:10]:
```suggestion
```

**Review by jacquelynsmale — DISMISSED** [review]
Since we haven't released 0.6.0 yet, I think this can still be a part of that release!

**Review by jacquelynsmale — DISMISSED** [review]

**Review by jacquelynsmale — APPROVED** [review]

**Review by jacquelynsmale — APPROVED** [review]

**Review by jacquelynsmale — APPROVED** [review]

**Pull request merged**